packages/orbit-components/src/InputField/InputTags/index.js

```diff
@@ -18,7 +18,6 @@ const StyledInputTags = styled.div`
   z-index: 2;
   min-width: 50px;
   overflow: hidden;
-  order: -1;
```

Comment:
Do you remember why you put this there in the first place?

Reply:
not sure, it's too old. Looks like legacy thing from the previous PRs